scripts/calculateKpi/protocols/tetherV0/alignMagicPrefixAndPadHex.ts

```diff
@@ -0,0 +1,24 @@
+import type { Hex } from 'viem'
+import { DIVVI_MAGIC_PREFIX } from './parseReferral'
```

Comment:
idk if we care about require cycles but it is more obvious that we have one here now 😅